packages/python-packages/doc-warden/warden/cmd_entry.py

```diff
@@ -76,39 +110,96 @@ def output_content_results(readmes_with_issues, config):
+        print()
+
+    if len(empty_release_notes):
+        print('{0} {1} empty release note for the latest package version'.format(len(empty_release_notes), pluralize('changelog has', 'changelogs have', len(empty_release_notes))))
+        print()
+        for changelog_tuple in empty_release_notes:
+            print('EMPTY CHANGELOG ENTRY: Latest Version {0} has no release notes in {1}. Consider adding release notes'.format(changelog_tuple[1]['curr_pkg_version'], changelog_tuple[0]))
+        print()
+
+# print presence results
+def output_presence_results(missing_target_file_paths, config):
+    if len(missing_target_file_paths):
+        print('{0} missing {1}{2} detected at:'.format(len(missing_target_file_paths), config.target_files[0], 's' if len(missing_target_file_paths) > 1 else ''))
+        for path in missing_target_file_paths:
             print(config.get_output_path(path))
         print()
 
+# print case issues
+def output_case_results(readmes_with_wrong_case, changelogs_with_wrong_case):
+    if len(readmes_with_wrong_case):
+        print('{0} Readme{1} are wrongly named:'.format(len(readmes_with_wrong_case), 's' if len(readmes_with_wrong_case) > 1 else ''))
+        for path in readmes_with_wrong_case:
+            print(path)
+        print()
+
+    if len(changelogs_with_wrong_case):
+        print('{0} Changelog{1} are wrongly named:'.format(len(changelogs_with_wrong_case), 's' if len(changelogs_with_wrong_case) > 1 else ''))
+        for path in changelogs_with_wrong_case:
+            print(path)
+        print()
+
+
 # execute both presence and content verification
 def all_operations(config):
+    packages = index_packages(config)
 
     if config.verbose_output:
-        print('Starting Content Examination')
-    content_results, ignored_content_results = verify_readme_content(config)
+        print('Starting Readme Presence Examination')
 
+    readme_presence_results, ignored_readme_presence_results = find_missing_target_files(config)
     if config.verbose_output:
-        print('Done Content Examination')
-        print('Starting Presence Examination')
-    presence_results, ignored_presence_results = find_missing_readmes(config)
+        print('Done with Readme Presence Examination')
+        print('Starting Readme Content Examination')
+
+    readme_content_results, ignored_readme_content_results = verify_readme_content(config)
     if config.verbose_output:
-        print('Done Presence Examination')
+        print('Done with Readme Content Examination')
 
-    output_presence_results(presence_results, config)
-    output_content_results(content_results, config)
+    output_presence_results(readme_presence_results, config)
+    output_readme_content_results(readme_content_results, config)
+
+    config.target_files = ['changelog.md'] if config.target_files[0] == 'readme.md' else ['readme.md']
```

Comment:
What is happening here? If we tell the application to verify_readmes and we've provided a target_file of `changelog`, then I expect it to _do nothing_. I'd like to discuss this one a bit more.

Reply:
Hmm. I see now. You're doing this because you explicitly are doing `all_operations` here. Aka, do BOTH changelog and readme.

I see the conflict here, and I think we should take a step back and give a bit more thought on how we design this...feeling hacky.